cmake/projects/FP16/hunter.cmake

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2016-2019, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cmake_args)
+
+hunter_add_version(
+    PACKAGE_NAME
+    FP16
+    VERSION
+    0.0.0-febbb1c-p1
```

Comment:
Should be `0.0.0-febbb1c-p0` instead of `0.0.0-febbb1c-p1`.

Reply:
Sorry, was a typo when I created the release in my branch. Forgot to fix.